test/fakeintake/server/http.go

```diff
@@ -22,3 +25,36 @@ func writeHttpResponse(w http.ResponseWriter, response httpResponse) {
 		w.Write(response.body)
 	}
 }
+
+func buildErrorResponse(responseError error) httpResponse {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏I believe this should never be given a `nil` parameter so it's possible to simplify it

Reply:
You're right ! Fixed in 411fe3bf4c360d94335f78878775862550d3686f